postgres-appliance/Dockerfile

```diff
@@ -62,27 +62,33 @@ RUN export DEBIAN_FRONTEND=noninteractive \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/* /root/.cache
 
+
 # install etcdctl
 ENV ETCDVERSION 2.3.3
 RUN curl -L https://github.com/coreos/etcd/releases/download/v${ETCDVERSION}/etcd-v${ETCDVERSION}-linux-amd64.tar.gz \
     | tar xz -C /bin --strip=1 --wildcards --no-anchored etcdctl
 
+# install supervisord
+RUN    apt-get update \
+    && apt-get upgrade -y \
```

Comment:
Why do we need apt-get update? That may install quite a lot of things we don't really need.